appmgr/src/version/v0_2_8.rs

```diff
@@ -0,0 +1,36 @@
+use super::*;
+use crate::util::Invoke;
+
+const V0_2_8: emver::Version = emver::Version::new(0, 2, 8, 0);
+
+pub struct Version;
+#[async_trait]
+impl VersionT for Version {
+    type Previous = v0_2_7::Version;
+    fn new() -> Self {
+        Version
+    }
+    fn semver(&self) -> &'static emver::Version {
+        &V0_2_8
+    }
+    async fn up(&self) -> Result<(), Error> {
+        for (app_id, _) in crate::apps::list_info().await? {
+            tokio::process::Command::new("docker")
+                .arg("stop")
+                .arg(&app_id)
+                .invoke("Docker")
+                .await?;
+            tokio::process::Command::new("docker")
+                .arg("update")
+                .arg("--restart")
+                .arg("no")
+                .arg(&app_id)
+                .invoke("Docker")
+                .await?;
+        }
+        Ok(())
+    }
+    async fn down(&self) -> Result<(), Error> {
+        Ok(())
+    }
```

Comment:
is this down required by the structure of the file? Because this down isn't legit, but we also don't support downs in general right now.

Reply:
the down is valid. the up corrects a mistake that was made in 0.2.7, but if you were to revert to 0.2.7 you wouldn't want to undo it.